src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -1953,7 +1953,7 @@ void PolicyHandler::OnPTUFinished(const bool ptu_result) {
 }
 
 bool PolicyHandler::CanUpdate() {
-  return 0 != GetAppIdForSending();
+  return true;
```

Comment:
Instead of changing the CanUpdate() behavior to return true, I would omit calling the function where applicable.

Reply:
Agreed, that seems like a much better approach. Fixed in https://github.com/smartdevicelink/sdl_core/pull/3280/commits/4bb0606f3bede5abd33b28cd1d8f89d3a9b29ecd